controllers/workspace_controller_runs.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package controllers
+
+import (
+	"context"
+	"fmt"
+
+	tfc "github.com/hashicorp/go-tfe"
+	appv1alpha2 "github.com/hashicorp/terraform-cloud-operator/api/v1alpha2"
+)
+
+func (r *WorkspaceReconciler) reconcileRuns(ctx context.Context, w *workspaceInstance, workspace *tfc.Workspace) error {
+	w.log.Info("Reconcile Runs", "msg", "new reconciliation event")
+
+	if workspace.CurrentRun == nil {
+		w.log.Info("Reconcile Runs", "msg", "there is no current run")
```

Comment:
I take it this means "there is no run currently in progress"? 

I'd be nice to have more user friendly error messages in the log.
IMO, to a human user looking at the logs "current run" isn't a very precise indication without knowing the API convention for the actively executing Run is "current run".

Reply:
There are two types of runs: normal run and speculative plan run. Rephrased it to be 'there are currently no ongoing runs that are not speculative'. Speculative plan runs will be covered in different PR since it is a new feature.

Thanks!